esp/src/src-react/components/forms/landing-zone/BlobImportForm.tsx

```diff
@@ -207,7 +213,7 @@ export const BlobImportForm: React.FunctionComponent<BlobImportFormProps> = ({
                                     rules={{
                                         required: nlsHPCC.ValidationErrorTargetNameRequired,
                                         pattern: {
-                                            value: /^(?:[\/\\]?[-a-z0-9_]+[\/\\])+([-a-z0-9 _\.]+)$/i,
+                                            value: /^(?:[\/\\]?[-a-z0-9_]+)+([-a-z0-9 _\.]+)$/i,
```

Comment:
The filename should not contain "\\" or "/"?

Reply:
It should, but only here, as Dan pointed out in HPCC-28222. But I copied & pasted the regex into the other forms by mistake. Fixed in those.

Reply:
@jeclrsg this filename could contain "\" or "/"?

Reply:
@GordonSmith No, I'd realized that second, non-optional directory separator there was incorrect. Testing various path-y type strings on https://regex101.com